### ISSUE TITLE
[Feature Request] Add `MS-SSIM + L1` reconstruction loss

### ISSUE THREAD

**Issue by hasibzunair** [kornia/kornia#1542]
## 🚀 Feature

Implementation of MS-SSIM + L1 loss function which can be used in vision tasks such as image inpainting, restoration, synthesis etc.

## Motivation

While there is an implementation of [SSIM loss](https://kornia.readthedocs.io/en/latest/losses.html#kornia.losses.SSIM) in `korina.losses`, there isn't one for MS-SSIM + L1. It is annoying to always use a big function with my other loss terms!

Have a look at the paper [Loss Functions for Image Restoration with Neural
Networks](https://research.nvidia.com/sites/default/files/pubs/2017-03_Loss-Functions-for/NN_ImgProc.pdf#page11) which shows MS-SSIM + L1 is simply awesome.

Related implementations are [here](https://github.com/NVlabs/PL4NN) in Caffe. And, an implementation of MS-SSIM [here](https://github.com/VainF/pytorch-msssim) in PyTorch.

## Pitch

I want to use MS-SSIM + L1 loss in one-ish line(s) of code in my projects.
______________________________________________________________________

**Comment by edgarriba** [comment]
@hasibzunair Thanks for pointing this out. Are you willing to give it a shot ?
Besides, I believe this is duplicated from https://github.com/kornia/kornia/issues/451
Also worth to check on https://github.com/kornia/kornia/issues/1158

**Comment by hasibzunair** [comment]
@edgarriba, yup. Should I get started with the MS-SSIM + L1 then? Can see others working on similar features!

**Comment by edgarriba** [comment]
sounds like a plan

**Comment by hasibzunair** [comment]
@edgarriba  How do I run https://kornia.readthedocs.io/en/latest/losses.html locally to see the new changes?

**Comment by edgarriba** [comment]
I suggest to read this: https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst#developing-kornia
And then once you modify (inside the dev-env) -- `pytest test/losses/test_losses.py::TestSSIMLoss::test_ssim`
https://github.com/kornia/kornia/blob/master/test/losses/test_losses.py#L301

**Comment by hasibzunair** [comment]
Is there a way to see the documentation locally too? I read https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst#developing-kornia but didn't see anything on that. Not sure if I am missing something. Wanna double check the math equations hahah.

**Comment by edgarriba** [comment]
that one is missing, you're right (worth adding :) -- you should `make build-docs`
https://github.com/kornia/kornia/blob/master/Makefile#L50
